packages/@ourworldindata/utils/src/owidTypes.ts

```diff
@@ -690,6 +691,76 @@ export type EnrichedBlockSDGGrid = {
     items: EnrichedSDGGridItem[]
 } & EnrichedBlockWithParseErrors
 
+export type RawBlockStickyRightContainer = {
+    type: "sticky-right"
+    value: {
+        left: OwidRawArticleBlock[]
+        right: OwidRawArticleBlock[]
+    }
+}
+
+export type EnrichedBlockStickyRightContainer = {
+    type: "sticky-right"
+    left: OwidEnrichedArticleBlock[]
+    right: OwidEnrichedArticleBlock[]
+} & EnrichedBlockWithParseErrors
+
+export type RawBlockStickyLeftContainer = {
+    type: "sticky-left"
+    value: {
+        left: OwidRawArticleBlock[]
+        right: OwidRawArticleBlock[]
+    }
+}
+
+export type EnrichedBlockStickyLeftContainer = {
+    type: "sticky-left"
+    left: OwidEnrichedArticleBlock[]
+    right: OwidEnrichedArticleBlock[]
+} & EnrichedBlockWithParseErrors
+
+export type RawBlockSideBySideContainer = {
+    type: "side-by-side"
+    value: {
+        left: OwidRawArticleBlock[]
+        right: OwidRawArticleBlock[]
+    }
+}
+
+export type EnrichedBlockSideBySideContainer = {
+    type: "side-by-side"
+    left: OwidEnrichedArticleBlock[]
```

Comment:
This is fine for now but longer term I don't think we really want to allow arbitrary recursion here and in many other places. For a lot of containers, only a subset of elements really makes sense and I think it would be nice to express this on the time. As I said, not for now, but just to communicate this line of thinking